recipes/libs/openssl.yaml

```diff
@@ -13,24 +13,68 @@ depends:
 checkoutSCM:
     scm: url
     url: https://www.openssl.org/source/openssl-${PKG_VERSION}.tar.gz
-    digestSHA256: 1e3a91bc1f9dfce01af26026f856e064eab4c8ee0a8f457b5ae30b40b8b711f2
+    digestSHA256: e8be6a35fe41d10603c3cc635e93289ed00bf34b79671a3a4de64fcee00d5242
     extract: False
 
 buildTools: [target-toolchain]
-buildVars: [PKG_VERSION, CC, AR, RANLIB, ARCH, AUTOCONF_BUILD, AUTOCONF_HOST]
+buildVars: [PKG_VERSION, CC, AR, RANLIB, ARCH, AUTOCONF_BUILD, AUTOCONF_HOST, BASEMENT_LIBS]
 buildScript: |
     export TARGETMACH=${AUTOCONF_HOST}
-    export LDFLAGS="-Wl,-rpath-link=${BOB_DEP_PATHS[libs::zlib-dev]}/usr/lib"
```

Comment:
the rpath stuff isn't a good idea for being relocatable, is it?

Reply:
The rpath-link does not end up in the binary. It's there to find transitive shared library dependencies. The cpackage class normally takes care of this and could indeed be used here. Did you test if it still builds in all cases after removing this line?

https://stackoverflow.com/questions/24598047/why-does-ld-need-rpath-link-when-linking-an-executable-against-a-so-that-needs

Reply:
Not removed anymore, just added to the configure directly.